config.rb.sample

```diff
@@ -6,12 +6,17 @@ $new_discovery_url='https://discovery.etcd.io/new'
 #if File.exists?('user-data') && ARGV[0].eql?('up')
 #  require 'open-uri'
 #  require 'yaml'
-# 
+#
 #  token = open($new_discovery_url).read
```

Comment:
Can you just move the assignment `$num_instances=1` before this and change this to `$new_discovery_url='https://discovery.etcd.io/new?size=%d' % [num_instances]` (I think this is the correct syntax)?